test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -563,10 +585,10 @@ void testBCTest(json_spirit::mObject& _o)
 	ImportTest::compareStates(postState, blockchain.topBlock().state());
 }
 
-bigint calculateMiningReward(u256 const& _blNumber, u256 const& _unNumber1, u256 const& _unNumber2)
+bigint calculateMiningReward(u256 const& _blNumber, u256 const& _unNumber1, u256 const& _unNumber2, bool isByzantium)
 {
 	unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(test::TestBlockChain::s_sealEngineNetwork)).createSealEngine());
-	bigint baseReward = se->chainParams().blockReward;
+	bigint baseReward = se->chainParams().blockReward(isByzantium);
```

Comment:
why? chainParams initialized from genesisInfo. it should know that. cause seal engine knows that from chainParams.

``` chainParams().u256Param("byzantiumForkBlock")  ```  
indicates when Byzanium starts.  

do you think we should look at this issue more carefully and perhaps do a refactoring PR for libethereum instead?